changedetectionio/processors/restock_diff.py

```diff
@@ -39,6 +56,24 @@ def run_changedetection(self, uuid, skip_when_checksum_same=True):
 
         # Main detection method
         fetched_md5 = None
+
+        # Try/prefer the structured data first if it exists
+        # https://schema.org/ItemAvailability Which strings mean we should consider it in stock?
+        availability = self.get_itemprop_availability()
+        if availability:
+            if any(availability in s for s in
+                   [
+                       'InStock',
+                       'InStoreOnly',
+                       'LimitedAvailability',
+                       'OnlineOnly',
+                       'PreSale' # Debatable?
+                   ]):
+                self.fetcher.instock_data = 'Possibly in stock'
+            else:
+                self.fetcher.instock_data = availability
```

Comment:
This will trigger a change for people where the item is not available, and they are using the previous "scraping output" hmm, not sure what todo here